npm/private/npm_translate_lock_state.bzl

```diff
@@ -137,20 +137,26 @@ def _init_update_labels(rctx, label_store):
         label_store.add("yarn_lock", attr.yarn_lock, seed_root = True)
 
 ################################################################################
-def _init_patch_labels(rctx, label_store):
+def _init_patch_labels(priv, rctx, label_store):
     if rctx.attr.verify_patches:
         label_store.add("verify_patches", rctx.attr.verify_patches)
 
     patches = []
     for pkg_patches in rctx.attr.patches.values():
         patches.extend(pkg_patches)
 
+    # Add patches from `pnpm.patchedDependencies`
+    for patch_info in priv["patched_dependencies"].values():
+        patches.append("//:%s" % patch_info.get("path"))
+
```

Comment:
There is a bug here that I haven't solved. If you rename a patch file and try to update the lockfile with `update_pnpm_lock = True`, then this will fail because the lockfile is referencing a patch file that no longer exists, and the lockfile won't get updated.

Reply:
Is one solution to parse these from `package.json` instead?

Reply:
This is needed in order to precompute labels for the `verify_patches` feature.

Reply:
Fixed this by reading patches from package.json.